### ISSUE TITLE
the Wrong word

### ISSUE THREAD

**Issue by fei-GitHub** [EOSIO/eos#3265]
"Verison 1.1" should be "Version 1.1"

**Comment by bean5** [comment]
I think this can be closed. GitHub search does not reveal any string matching `Verison` in this project: https://github.com/search?utf8=%E2%9C%93&q=org%3AEOSIO+Verison&type=

**Comment by Velua** [comment]
Agreed, my recursive grep against the repo didn't yield anything either.
